lib/swaggerValidator/ajvSchemaValidator.ts

```diff
@@ -192,6 +201,49 @@ export const ajvErrorToSchemaValidateIssue = (
   return result;
 };
 
+const ReValidateIfNeed = (
+  errors: ErrorObject[],
+  ctx: SchemaValidateContext,
+  data: any,
+  validate: ValidateFunction
+): ErrorObject[] => {
+  const result: ErrorObject[] = [];
+  const length = errors.length;
+  const newData = lodash.cloneDeep(data);
+
+  for (let i = 0; i < length; i++) {
+    validate.errors = null;
+    const error = errors[i];
+    const { schema, parentSchema: parentSch, keyword, data: errorData, dataPath } = error;
+    const parentSchema = parentSch as Schema;
+
+    // If the value of query parameter is in string format, we can skip this error
+    if (
+      !ctx.isResponse &&
+      keyword === "type" &&
+      schema === "array" &&
+      typeof errorData === "string" &&
+      (parentSchema as any)?.["in"] === "query"
+    ) {
+      const arrayData = errorData.split(",").map((item) => {
+        if (parseFloat(item)) {
```

Comment:
Can you catch exception when item is not number?

Reply:
In addition to number and string, there is boolean that needs to be revalidated, revalidation has been added